developers/weaviate/search/generative.md

```diff
@@ -18,17 +18,70 @@ Retrieval Augmented Generation (RAG) combines information retrieval with generat
 
 In Weaviate, a RAG query consists of two parts: *a search query*, and a *prompt for the model*. Weaviate first performs the search, then passes both the search results and your prompt to a generative AI model before returning the generated response.
 
+## Configure a generative model provider
+
+:::info Added in `v1.30`
+:::
+
+To use RAG, [a collection must be configured](../manage-data/collections.mdx#specify-a-generative-model-integration) to use a [generative model integration](../model-providers/index.md) or you can spcify the generative model in the search operation directly:
```

Comment:
I like the change in principle. The word 'must' being carried over is a bit confusing as to what is optional imo. Wdyt about:

To use RAG with a [generative model integration](../model-providers/index.md): 
- [Set a default configuration for the collection](../manage-data/collections.mdx#specify-a-generative-model-integration), and/or 
- Provide the settings at the search operation: